docs/velox-backend-support-progress.md

```diff
@@ -88,400 +86,81 @@ Gluten supports 30+ operators (Drag to right to see all data types)
-| spark_partition_id           |                        |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| stack                        |                        |                       | S      |                          | S       | S    | S     | S   | S    | S     | S      | S    | S         | S      | S       | S    | S      | S        | S     | S   | S      | S   |
-| xxhash64                     | xxhash64               | xxhash64              |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| uuid                         | uuid                   | uuid                  | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| rand                         | rand                   | rand                  | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| try_add                      |                        |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| try_substract                |                        |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| try_multiply                 |                        |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| try_divide                   |                        |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-
+## Function Support Status
+
+Spark divides the built-in functions into four categories: Scalar Functions, Aggregate Functions, Window Functions and Generator Functions.
+The support list for each catagory is maintained in separate files. Please check the links below.
+
+[Scalar Functions Support Status](./velox-backend-scalar-function-support.md)
```

Comment:
Can you put the command here to generate the support md file? or mentioned in another readme?